modP2pImpl/test/org/aion/p2p/impl/comm/PeerMetricTest.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (c) 2017-2018 Aion foundation.
+ *      This file is part of the aion network project.
+ *
+ *      The aion network project is free software: you can redistribute it
+ *      and/or modify it under the terms of the GNU General Public License
+ *      as published by the Free Software Foundation, either version 3 of
+ *      the License, or any later version.
+ *
+ *      The aion network project is distributed in the hope that it will
+ *      be useful, but WITHOUT ANY WARRANTY; without even the implied
+ *      warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *      See the GNU General Public License for more details.
+ *
+ *      You should have received a copy of the GNU General Public License
+ *      along with the aion network project source files.
+ *      If not, see <https://www.gnu.org/licenses/>.
+ *
+ *  Contributors:
+ *      Aion foundation.
+ */
+
+package org.aion.p2p.impl.comm;
+
+
```

Comment:
formatting issue: two empty lines

Reply:
fixed in https://github.com/aionnetwork/aion/pull/588/commits/e7a741dac8c70c0feceaeec0a5ee6299e5cfa74d